Yggdrasil.Scripting/ScriptLoader.cs

```
@@ -343,44 +407,20 @@ private Assembly Compile(IEnumerable<string> scriptFilesList)
 
 				foreach (var location in toReference)
 				{
-					parameters.ReferencedAssemblies.Add(location);
+					AddAssembly(location);
 				}
-
-				// Compile, throw if compilation failed
-				var result = _compiler.CompileAssemblyFromFile(parameters, filePaths);
-				var errors = result.Errors;
-
-				if (errors.Count != 0)
+				var trees = new List<SyntaxTree>();
+				foreach (var sourceFile in filePaths)
 				{
-					foreach (CompilerError error in errors)
-					{
-						if (mapFilePaths.TryGetValue(error.FileName, out var fileName))
-							error.FileName = fileName;
-					}
-
-					throw new CompilerErrorException(errors);
```

It appears like you removed all forwarding of exceptions or errors and are simply swallowing them now. This is a no-go, as the user needs to know that the compilation failed and why. You can see in Melia how we use this information to print out errors in scripts.